app/Helpers/render/leaderboard.php

```diff
@@ -468,13 +469,27 @@ function getGlobalRankingData(
         if ($info == 0) {
             if ($unlockMode == UnlockMode::Hardcore) {
                 $selectQuery = "SELECT ua.User,
-                        (SELECT COALESCE(SUM(CASE WHEN HardcoreMode = " . UnlockMode::Hardcore . " THEN 1 ELSE 0 END), 0) FROM Awarded AS aw WHERE aw.User = ua.User) AS AchievementCount,
+                        (SELECT COALESCE(SUM(CASE WHEN aw.HardcoreMode = " . UnlockMode::Hardcore . " 
+                                                   AND ach.Flags = " . AchievementType::OfficialCore . " 
```

Comment:
You should move this Flags out of the CASE for consistency.

Reply:
👍 Change applied in https://github.com/RetroAchievements/RAWeb/pull/1617/commits/75d1bb3c4f430f8e2f34fb58997b55d8dcba73c6